docs/3____physical_signal_abstraction.adoc

```diff
@@ -13,44 +16,37 @@ This section explains the different variable types used to carry network traffic
 
 In case multiplexed signals are present in a frame/PDU: all signals are present, but only the active signal according to the multiplex switch signal contains a valid value, all inactive variables have undefined values _[those values could even be outside their specified min-max range without fault]_.
 
 Generally, signal variables are optional, but they must either be all missing or all present to clearly indicate support for the signal abstraction layer.
 
-All signal variables are clocked to indicate when exactly they have been sent, see <<ClockVariableHighCut>>.
+All signal variables are clocked to exactly indicate when they have been sent, see <<ClockVariableHighCut>>.
 
 
 ==== Clock Variables [[ClockVariableHighCut]]
-
-#TODO AJ#: Add description of Event Mode handling here and how the countdown clock pre-schedules events into the future for performance reasons
-
-
 In order to use FMU input and output variables as transport layer for networks, input and output clocks are used.
 Such a clock is scheduled by the sending FMU to indicate the transmission of the corresponding frame or frames.
 
 The output clock type, periodic or aperiodic, is defined by the sending FMU.
 This allows the FMU (or better the exporting tool) to balance the accuracy and performance of its network communication:
 
  - While aperiodic clocks allow very accurate network simulations, frequently entering Event Mode will reduce the network simulation speed.
- - Using periodic clocks and buffering the frames to be transmitted reduces the number of Event Modes required and speeds up the simulation at the cost of simulation accuracy.
+ - Using periodic clocks reduces the number of Event Modes required and speeds up the simulation at the cost of simulation accuracy, because timing must be forced into a fixed grid and some intermediate values might not be transmitted.
  - One could use (structural) parameters to define the accuracy of aperiodic clocks, allowing control of the simulation accuracy and performance with the same FMU.
  - The input clocks shall be triggered clocks.
 
 The importer will then connect and merge output clock activations, even those of different clock types, and forwards them to the input clocks, as required by the network semantics.
 
-All signal variables are clocked variables.
 Signal variables belonging to frame `BusName::FrameName` are triggered by the clock `BusName::FrameName_Clock` and all these variables and their corresponding clock must share the same `causality` (`input` or `output`).
 
 === Network Description Files [[NetworkDescriptionFile,network description file]]
 
-#TODO AJ#: remove old requirements for network description files (not needed anymore): make it optional
-
 Standardized <<NetworkDescriptionFile,network description files>> can be shipped with each FMU to describe properties of signals and frames, such as signal units, frame composition and trigger conditions.
```

Comment:
optional

Reply:
At the beginning the text says "A signal variable carries the physical value of a network signal normally packaged inside a PDU or frame. The unit definition of the variable must match the one defined in the [network description file](https://modelica.github.io/fmi-ls-bus/main/#NetworkDescriptionFile)." I think at this point we should clarify that they are optional

Reply:
Done. First comment in recent commit, second suggestion earlier commit